### ISSUE TITLE
Telegram Bot API is rate limited, handle this by buffering messages

### ISSUE THREAD

**Issue by FruitieX** [FruitieX/teleirc#108]
Otherwise messages can be lost quite easily on busy chat rooms.

**Comment by FruitieX** [comment]
Bundling together IRC messages into one on Telegram is another good idea.

**Comment by Joshua1337** [comment]
Maybe use telegram-cli?

**Comment by Mikaela** [comment]
Wouldn't that require separate account for TeleIRC?

**Comment by FruitieX** [comment]
Indeed. Plus it's not a very clean solution. Teleirc used to be built on telegram-cli btw, but switched over to the bot API once it was available.
